Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.yml

```diff
@@ -0,0 +1,1901 @@
+          isArray: false
+          description: The maximum number of projects to return.
+          defaultValue: 50
+        - name: page
+          required: false
+          isArray: false
+          description: The specific result page to display.
+        - name: page_size
+          required: false
+          isArray: false
+          description: The number of records in a page.
+      outputs:
+        - contextPath: NSM.Sensors.ID
+          description: Sensor ID.
+          type: Number
+        - contextPath: NSM.Sensors.Name
+          description: Name of the sensor.
+          type: String
+        - contextPath: NSM.Sensors.model
+          description: Sensor model.
+          type: String
+        - contextPath: NSM.Sensors.Description
+          description: Sensor description.
+          type: String
+        - contextPath: NSM.Sensors.DomainID
+          description: ID of the domain to which this sensor belongs.
+          type: Number
+        - contextPath: NSM.Sensors.isFailOver
+          description: Whether the sensor is failover.
+          type: Boolean
+        - contextPath: NSM.Sensors.isNTBA
+          description: Whether there is Network Threat Behavior Analysis (NTBA).
+          type: Boolean
+        - contextPath: NSM.Sensors.isLoadBalancer
+          description: Whether the sensor is a load balancer.
+          type: Boolean
+        - contextPath: NSM.Sensors.SerialNumber
+          description: The sensor serial number.
+          type: Unknown
+        - contextPath: NSM.Sensors.SigsetVersion
+          description: Signature set version number applied to the sensor.
+          type: String
+        - contextPath: NSM.Sensors.DATVersion
+          description: DAT version of the sensor.
+          type: String
+        - contextPath: NSM.Sensors.SoftwareVersion
+          description: Sensor software version.
+          type: String
+        - contextPath: NSM.Sensors.LastSignatureUpdateTs
+          description: Last configuration download timestamp.
+          type: Date
+        - contextPath: NSM.Sensors.IPSPolicyID
+          description: Intrusion prevent system (IPS) policy ID applied to the sensor.
+          type: Number
+        - contextPath: NSM.Sensors.ReconPolicyID
+          description: Recon policy ID applied to the sensor.
+          type: Number
+        - contextPath: NSM.Sensors.LastModTs
+          description: Last modified timestamp.
+          type: Unknown
+        - contextPath: NSM.Sensors.IP Address
+          description: Sensor IP address.
+          type: String
+        - contextPath: NSM.Sensors.nsmVersion
+          description: Network Security Manager (NSM) version.
+          type: String
+        - contextPath: NSM.Sensors.capacity
+          description: Sensor capacity.
+          type: String
+        - contextPath: NSM.Sensors.isStack
+          description: Is stack.
+          type: Boolean
+        - contextPath: NSM.Sensorss.isStackMember
```

Comment:
In this context Path, NSM.Sensorss.isStackMember, should it be NSM.Sensors.isStackMember?

Reply:
yes, I fixed it and didn't push it yet